hips/hip-999.md

```diff
@@ -0,0 +1,105 @@
+---
+hip: 9999
+title: "Helm Org Maintainers and Deprecated Projects"
+authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+created: "2020-09-24"
+type: "process"
+status: "draft"
+---
+
+## Abstract
+
+This process proposal explains what happens to Helm organization maintainer slots when
+a project is deprecated.
+
+## Motivation
+
+The current governance documents state that org maintainers may be selected from any of
+the official Helm projects. However, as the Helm organization matures, some projects are
+being deprecated. The governance docs are unclear what happens to org maintainers from
+deprecated projects, and whether deprecated projects may hold org maintainer seats.
+
+## Rationale
+
+In the interest of keeping the Helm project thriving and alive, this HIP proposes that
+only non-deprecated projects are entitled to org maintainer seats.
+
+One project that is slated for deprecation is the charts repository, which previously had
+guaranteed org maintainers. This proposal updates that by splitting its designated spots
```

Comment:
The charts project does not have guaranteed spots. When the governance was created at the initial election time it was guaranteed spots. That was added in https://github.com/helm/community/commit/115752de1f27d6b48b0283cc4f1262b25add03ee#diff-bcef1ca4b17b5dcae6fd40f5d117cb21. The relevant part reads:

> To ensure diverse representation in the initial selection of Org Maintainers from all areas of the Helm org, the breakdown of Helm Org Maintainers group will be as follows:
> 
> * 3 Representatives from the Helm core project
> * 2 Representative from the Charts project
> * 2 Representatives from another Helm project
> 
> This accounts for a total of 7 initial Org Maintainers.

There are no longer any guarantees based on projects.